Source/ASImageNode.h

```diff
@@ -168,6 +168,15 @@ typedef UIImage * _Nullable (^asimagenode_modification_block_t)(UIImage *image);
  */
 @property (copy) NSString *animatedImageRunLoopMode;
 
+/**
+ * @abstract Seek to a play head position of an animated image.
+ *
+ * @param playHead The desired play head position. Must be in valid duration interval.
+ *
+ * @discussion Seeking must be performed on the main-thread.
+ */
+- (void)animatedImageSeekToPlayHeadPosition:(CFTimeInterval)playHead;
```

Comment:
Mind adding a bit about how this may not be performant because the underlying animated image caches forward?